content/en/docs/concepts/storage/volumes.md

```diff
@@ -975,6 +975,36 @@ Simultaneous writers are not allowed.
 See the [RBD example](https://github.com/kubernetes/examples/tree/master/volumes/rbd)
 for more details.
 
+#### RBD CSI migration {#rbd-csi-migration}
+
+{{< feature-state for_k8s_version="v1.23" state="alpha" >}}
+
+The `CSIMigration` feature for `RBD`, when enabled, redirects all plugin
+operations from the existing in-tree plugin to the `rbd.csi.ceph.com` {{<
+glossary_tooltip text="CSI" term_id="csi" >}} driver. In order to use this
+feature, the
+[CEPH CSI driver](https://github.com/ceph/ceph-csi)
+must be installed on the cluster and the `CSIMigration` and `CSIMigrationRBD`
+[feature gates](/docs/reference/command-line-tools-reference/feature-gates/)
+must be enabled.
+
+{{< note >}}
+
+Before enabling migration for RBD, a Kubernetes storage admin has to complete
+below prerequisites in their setup:
+
+* ceph csi driver ( rbd.csi.ceph.com) v3.5.0 or above has to be installed in the
+  cluster
+* considering the `clusterID` field is a required parameter for CSI driver for
+  its operations, but in-tree StorageClass has `monitors` field as a required
+  parameter, a Kubernetes storage admin has to create a clusterID based on the
+  monitors hash ( ex:`#echo -n
+  '<monitors_string>' | md5sum`) in the CSI config map and keep the monitors
+  under this clusterID configuration.
+* Also, if `adminId` in the intree storageclass is different than `admin`, the
```

Comment:
nits (doesn't block a merge):
```suggestion
* Also, if the value of `adminId` in the in-tree storage class is different from `admin`, the
```
or
```suggestion
* Also, if the value of `adminId` in the in-tree StorageClass is different from `admin`, the
```

depending on which is more accurate (I haven't looked).

Reply:
This has been addressed.